app/views/ctc/questions/account_number/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<% @main_question = t("views.ctc.questions.account_number.title") %>
+
+<% content_for :page_title, @main_question %>
+
+<% content_for :form_card do %>
+  <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long", autocomplete: "off" } do |f| %>
+    <h1 class="h2"><%= @main_question %></h1>
+
+    <div class="form-card__content">
+      <%= f.cfa_input_field(:account_number, t("views.ctc.questions.account_number.account_number"), type: :password) %>
```

Comment:
Was this asked to be a password field? I feel like most of the time I enter these details on websites it's in cleartext to start with; it's tough for me to enter long digits of numbers without seeing them as I type